manifests/03-rbac-role-ns-console.yaml

```diff
@@ -0,0 +1,72 @@
+kind: Role
```

Comment:
Split the `rbac` for `openshift-console` and `openshift-console-operator` into separate files to reduce confusion.